interp/notation_ops.ml

```diff
@@ -748,7 +762,7 @@ let alpha_rename alpmetas v =
   if alpmetas == [] then v
   else try rename_glob_vars alpmetas v with UnsoundRenaming -> raise No_match
 
-let add_env (alp,alpmetas) (terms,termlists,binders,binderlists) var v =
+let add_env (vars,(alp,alpmetas)) (terms,termlists,binders,binderlists) var v =
```

Comment:
Hasn't the time come to make `(vars,(alp,alpmetas))` a record? Give more explicit names would be useful.

Reply:
We could make a record, but I decided for minimal changes. Rephrasing with records can be done in a later phase.

Reply:
> Give more explicit names would be useful.

I don't find `alp` so explicit either. I may call it instead `renamings` or `alphasubst`. As for the `alpmetas`, it is the map between the binder name in the term and the (meta)name of the binder name in the pattern. It could be something like `binders`, or `bindersinst`?

As for the name `vars`, this is the name already used in `constrextern.ml`, so this provides some consistency.

BTW, if you'd like we have a more efficient interaction thanks e.g. to an online discussion, we may try.